components/atoms/PullRequestOverviewChart/pull-request-overview-chart.tsx

```diff
@@ -10,24 +10,29 @@ const PullRequestOverviewChartBar: React.FC<PullRequestOverviewChartBarProps & R
   type,
   ...event
 }) => {
+  const getBarColor = (type: string) => {
+    switch (type) {
+      case "open":
+        return "bg-light-grass-9";
+      case "merged":
+        return "bg-purple-600";
+      case "closed":
+        return "bg-light-red-9";
+      case "draft":
+        return "bg-light-slate-9";
+      default:
+        return "";
+    }
+  };
+
   return (
     <div
       {...event}
       className={`
-        ${
-          type === "open"
-            ? "bg-light-grass-9"
-            : type === "merged"
-            ? "bg-purple-600"
-            : type === "closed"
-            ? "bg-light-red-9"
-            : type === "draft"
-            ? "bg-light-slate-9"
-            : ""
-        }
-    ${percent === 0 && "hidden"}
+        ${getBarColor(type!)}
```

Comment:
Refactor (blocking): The `!` is not necessary. Avoid changing the type whenever possible. It looks like in the old code, it supported `undefined` as it returned `""` which is the default for the switch case now.

`const getBarColor = (type: string|undefined) => {...}` should do the trick.

Reply:
nit: Since this is not part of the fix, consider this for another PR as it's not related to the fix.